scss/_layouts_application.scss

```diff
@@ -94,16 +95,28 @@ $application-layout--side-nav-width-expanded: 15rem !default;
       }
     }
 
+    .l-navigation__drawer {
+      max-width: $application-layout--side-nav-width-expanded;
+    }
+
+    .l-navigation:hover,
+    .l-navigation.is-pinned {
+      overflow-y: auto;
+      width: $application-layout--side-nav-width-expanded;
+
+      .l-navigation__drawer {
+        max-width: 100%;
```

Comment:
@sowasred2012 You probably were right about the transition needing to be only on one direction. This seems to work for me. A bit hard to verify it via browserstack, but seems fine on windows as well. Can you check?

```suggestion
        max-width: 100%;
        transition: max-width 0ms map-get($animation-duration, fast);
```

Reply:
Yeah that fixes the transition back we were seeing, the only nagging thing is the slight layout shift when it opens:
![navbar](https://user-images.githubusercontent.com/2376968/101656273-6d3a8d00-3a3a-11eb-8feb-b645dfca11ea.gif)